test/core/test/__snapshots__/mocked.test.ts.snap

```diff
@@ -45,22 +45,22 @@ Received:
     1st spy call return:
 
     {
-  -   a: '4',
-  +   a: '1',
+  -   a: '1',
```

Comment:
This is now incorrect. "1" should be received

Reply:
`expect(a).toBe(b)` a is expected, b is received?

Reply:
It's not expected to be, it's what was called (expected) compared to what was received (actual)

The function was called with 1, but diff shows it was called with 4

Reply:
After this PR change it seems to become called(received) compared to what was expected, so this is correct.

Reply:
I don't know what you mean, but the diff here is incorrect. The function returned `{ a: '1' }`, but test expected `{ a: '4' }`. So the correct diff is:

```
- a: '4'
+ a: '1'
```

Reply:
I'm a little dizzy now 😵‍💫

According to your description, the tests seem to be reversed.

Reply:
With your change it is reversed, yes, but it shouldn't be.

Reply:
Sorry, I'm not sure what the correct modification is now. Maybe just reverse the colors here?
https://github.com/vitest-dev/vitest/blob/9836ccb48c5a39c0999484f55731f53ba494be17/packages/utils/src/descriptors.ts#L26-L27

Reply:
> Sorry, I'm not sure what the correct modification is now. Maybe just reverse the colors here?

Ok, I see now. Give me a bit more time to think